plaso/output/shared_elastic.py

```diff
@@ -182,7 +182,16 @@ def __init__(self, output_mediator):
     self._url_prefix = None
 
   def _Connect(self):
-    """Connects to an Elasticsearch server."""
+    """Connects to an Elasticsearch server.
+
+    Raises:
+      RuntimeError: if the Elasticsearch version is not supported or the server
+          cannot be reached.
+    """
+    if 6 < elasticsearch.__version__[0] < 8:
```

Comment:
IIUC, the only ES version which will trigger the RuntimeError is 7, while it looks like we want to support versions > 6

Reply:
good catch, changed.